### DISCUSSION TITLE
Additional Printer Columns

### DISCUSSION THREAD

**Discussion by somasundaram1996** [operator-framework/java-operator-sdk#1602] [Q&A] [answered]
Does java operator sdk provide option to include additional printer columns for crds generation

**Comment by csviri** [comment, marked as answer]
@somasundaram1996 the CRD generaion is from fabric8 client, see issue there:
https://github.com/fabric8io/kubernetes-client/issues/3011
AFAIK this is not implemented there yet. Might be worth to rather comment on this issue.
